client/spaces/client.go

```diff
@@ -23,6 +23,7 @@ type SpacesClient interface {
 	ListVirtualHostsByStage(spaceID, stage string) (VirtualHostList, error)
 	UpdateVirtualHost(spaceID, stage string, vhost VirtualHost) (*VirtualHost, error)
 	DeleteVirtualHost(spaceID, stage, hostname string) error
+	ListDatabaseUsersByStage(spaceID, stage string) (DatabaseUserList, error)
```

Comment:
Is this already complete? Or are there operations like `DeleteDatabaseUser` or similar still missing?

Reply:
`userData` is not designed to store users (and more importantly, passwords). User management should be done via a separate set of commands.  /cc @hensur